week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -26,7 +26,7 @@ function mood() {
 
 function greaterThan10() {
   const num = 10;
-  const isBigEnough;
+  const isBigEnough = true ;
```

Comment:
Well, this is right :), however, the aim would have been to use comparison operation to calculate the boolean value instead of using a baked in value. It could then reflect upon any changes in case content `num` changed.

Reply:
I think I'm puzzeled by this one, because isBigEnough was declared as a const and wasn't initialised, now if I change it to let and compare it in the if statement to num, it will throw an error as it holds no value. and the instructions says that i can only change isBigEnough. 
to use a comparison operator i'll need a third variable or input to assign the result of the comparison to isBigEnough. for example :
`  let input = 20
  const num = 10;
  const isBigEnough =(input > num) ;`
I can't really think of anyother way.

Reply:
I've thought of something like this:
`consts isBigEnough = num >= 10`

You don't need to use a third variable, you can use any constant value in your expression.

Reply:
But I agree that this exercise is a bit flaky.